torch_uncertainty/routines/classification.py

```diff
@@ -205,8 +217,14 @@ def criterion(self) -> nn.Module:
             self.loss = partial(self.loss, model=self.model)
         return self.loss()
 
-    def forward(self, inputs: Tensor) -> Tensor:
-        return self.model.forward(inputs)
+    def forward(self, inputs: Tensor, return_features: bool = False) -> Tensor:
+        if return_features:
+            features = self.model.feats_forward(inputs)
+            logits = self.model.linear(features)
```

Comment:
the last layer must be implicitly called "linear", we could also ask for a "classification_head" nn.Module in networks when we want to compute the grouping loss